libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/dh-market-participant-edit-actor.component.ts

```diff
@@ -64,13 +57,13 @@ export class DhMarketParticipantEditActorComponent {
       actorId: params[dhMarketParticipantActorIdParam] as string,
     }))
   );
+  triggerValidationSubject = new Subject<void>();
```

Comment:
nitpick: delete this property since it's not used.

Reply:
fixed